draft-reddy-tls-composite-mldsa.md

```diff
@@ -131,6 +136,13 @@ defined in {{I-D.ietf-lamps-pq-composite-sigs}}.
 
 The schemes defined in this document MUST NOT be used in TLS 1.2 {{RFC5246}}. A peer that receives ServerKeyExchange or CertificateVerify message in a TLS 1.2 connection with schemes defined in this document MUST abort the connection with an illegal_parameter alert.
 
+# Signature Algorithm Restrictions
+
+TLS 1.3 removed support for RSASSA-PKCS1-v1_5 {{RFC8017}} in CertificateVerify messages, opting for RSASSA-PSS instead. Similarly, this document restricts the use of the composite signature algorithms mldsa44_rsa_pkcs1_sha256 and mldsa65_rsa_pkcs1_sha384 to the "signature_algorithms_cert" extension. These composite signature algorithms MUST NOT be used with the "signature_algorithms" extension.
+These values refer solely to signatures which appear in certificates (see {{Section 4.4.2.2 of RFC8446})} and are not defined for use in signed TLS handshake messages.
+
+To enforce these rules, a violation of the expected usage of these signature algorithms will result in the transmission of a "illegal_parameter" alert.
```

Comment:
this is too vague

"A peer that receives a CertificateVerify message that indicates use of RSASSA-PKCS1-v1_5 algorithm MUST reject the connection with a fatal `illegal_parameter` Alert."

Reply:
I slightly modified the text as follows:
A peer that receives a CertificateVerify message indicating the use of the RSASSA-PKCS1-v1_5 algorithm as one of the component signature algorithms MUST terminate the connection with a fatal illegal_parameter alert.

Reply:
yup, that's even better, thanks!